### ISSUE TITLE
[Bug]: Empty input field on edit page

### ISSUE THREAD

**Issue by wotta** [ariaieboy/filament-currency#7]
### What happened?

When you have the following input field like the one below adding data in the form works as expected. When editing a record the input stays empty.

```php
TextInput::make('balance')
    ->prefix('€')
    ->currencyMask()
    ->required(),
```

### How to reproduce the bug

Use the following field it should happen.

```php
TextInput::make('balance')
    ->prefix('€')
    ->currencyMask()
    ->required(),
```

### Package Version

1.1.2

### PHP Version

8.2.8

### Laravel Version

10.25.2

### Which operating systems does with happen with?

Windows

### Notes

I am using WSL version: 2.0.4.0

**Comment by ariaieboy** [comment]
fixed in [1.1.3](https://github.com/ariaieboy/filament-currency/releases/tag/1.1.3)

**Comment by wotta** [comment]
Thanks, @ariaieboy! I tried to find the issue but do not know about filament plugins.
Really appreciate the quick fix! Have a nice day
